core/src/components/popover-menu/popover-menu-item/popover-menu-item.tsx

```diff
@@ -0,0 +1,26 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'tds-popover-menu-item',
+  styleUrl: 'popover-menu-item.scss',
+  shadow: true,
+})
+export class TdsPopoverMenuItem {
+  /** Disables the Popover Menu Item */
+  @Prop() disabled: boolean = false;
+
+  render() {
+    return (
+      <Host role="listItem">
```

Comment:
💯 i believe the role should be all lowercase: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role

Reply:
You are correct sir. Not sure why we have used it camelCased everywhere else?

Reply:
Hmm, seems like a mistake that caught on